Makefile

```diff
@@ -13,6 +13,7 @@ OUTPUT_BINARY ?= "$(BIN_DIR)/$(OPERATOR_NAME)"
 VERSION_PKG ?= "github.com/jaegertracing/jaeger-operator/pkg/version"
 JAEGER_VERSION ?= "$(shell grep -v '\#' jaeger.version)"
 OPERATOR_VERSION ?= "$(shell git describe --tags)"
+STORAGE_NAMESPACE ?= "storage"
```

Comment:
Note that this breaks our examples. The storage URLs in examples are `elasticsearch:9200`...

I would rather get the current namespace if no specified.